onnxruntime/core/providers/rocm/math/softmax.h

```diff
@@ -20,14 +20,14 @@ Status SoftMaxComputeHelper(
     int64_t axis,
     RocmTuningContext* tuning_ctx = nullptr);
 
-template <typename InputT, typename OutputT, typename AccT, bool IsLogSoftmax>
-Status dispatch_warpwise_softmax_forward(Stream* stream, OutputT* dst, const InputT* src, int softmax_elements,
-                                         int softmax_elements_stride, int batch_count,
+template <typename input_t, typename output_t, typename acc_t, bool is_log_softmax>
+Status dispatch_warpwise_softmax_forward(Stream* stream, output_t* dst, const input_t* src,
+                                         int softmax_elements, int softmax_elements_stride, int batch_count,
                                          RocmTuningContext* tuning_ctx = nullptr);
 
-template <typename InputT, typename OutputT, typename AccT, bool IsLogSoftmax>
-Status dispatch_blockwise_softmax_forward(Stream* stream, OutputT* output, const InputT* input, int softmax_elements,
-                                          int input_stride, int output_stride, int batch_count,
+template <typename input_t, typename output_t, typename acc_t, bool is_log_softmax>
+Status dispatch_blockwise_softmax_forward(Stream* stream, output_t* output, const input_t* input,
+                                          int softmax_elements, int input_stride, int output_stride, int batch_count,
```

Comment:
These changes are unnecessary and it contains outrageously annoying style mismatch.

Reply:
Fixed by cfd74fc6d64e5ac8fcae05874d1a7ecb79e2e327 and 2afe507a5f46c73bcf6b706a8a02a4133242c155.